templates/quickstart-jira-dc.template.yaml

```diff
@@ -1303,6 +1316,23 @@ Resources:
     Properties:
       AliasName: !Sub "alias/${AWS::StackName}"
       TargetKeyId: !Ref EncryptionKey
+
+# Optional: Cloudwatch dashboard to be created when CloudWatch is enabled
+  CloudwatchDashboard:
+    DependsOn:
+      - DB
+    Condition: EnableCloudWatch
+    Type: AWS::CloudFormation::Stack
+    Properties:
+      TemplateURL: !Sub
+        - https://${QSS3BucketName}.${QSS3Region}.amazonaws.com/${QSS3KeyPrefix}submodules/quickstart-atlassian-services/templates/quickstart-cloudwatch-dashboard.yaml
```

Comment:
The current PR with the dashboard contains log messages. What will happen if user doesn't select `Metrics and Logs` but just `Metrics Only`?

Reply:
@tarka and I spoke about this case earlier. Here's a summary of our conversation - 

When this happens, users will see a dashboard with metrics, but no logs. The dashboard won't break, it'll just not be populated by any logs.

As the dashboard is meant to be a starting point and not a recommended reference, users will end up tweaking the metrics that they care. We figured it's not worth taking care of both the cases

Reply:
Sounds good, the main thing is that the board is not broken. Also when people change the metrics settings, it will probably make the logs visible on already existing template  🤞